app/Console/Commands/InstallCommand.php

```diff
@@ -109,9 +111,9 @@ protected function configureDatabase(array $default = [])
         ], $default);
 
         $config['DB_DRIVER'] = $this->choice('Which database driver do you want to use?', [
-            'mysql'      => 'MySQL',
-            'postgresql' => 'PostgreSQL',
-            'sqlite'     => 'SQLite',
+            'mysql'  => 'MySQL',
+            'pgsql'  => 'PostgreSQL',
```

Comment:
Instead of doing that, best to just hard code the 3 drivers we support, instead of calling getAvailableDrivers, because if laravel were to add a new driver, we wouldn't actually support it, so, best not to write code that says we do. :)